server/src/modules/organizations/organizations.module.ts

```diff
@@ -25,6 +25,17 @@ import { AppGroupPermission } from 'src/entities/app_group_permission.entity';
 import { UserGroupPermission } from 'src/entities/user_group_permission.entity';
 import { EncryptionService } from '@services/encryption.service';
 import { OrgEnvironmentVariablesService } from '@services/org_environment_variables.service';
```

Comment:
try removing `OrgEnvironmentVariablesService` import, other imports won't be required

Reply:
that is not working